client-cli/src/command/wallet_command.rs

```diff
@@ -31,6 +37,43 @@ pub enum WalletCommand {
         )]
         wallet_type: WalletKind,
     },
+    #[structopt(name = "export", about = "Backup wallet to a file")]
+    Export {
+        #[structopt(
+            name = "wallet name",
+            short = "n",
+            long = "name",
+            help = "Name of wallet"
+        )]
+        name: Option<String>,
+        #[structopt(
+            name = "from_file",
+            short = "f",
+            long = "from_file",
```

Comment:
how about just form

Reply:
I think it's ok -- if it's too long, one can just use `-f ... -t ...`?

Reply:
Ok